packages/wrangler/src/create-worker-upload-form.ts

```diff
@@ -257,7 +258,7 @@ export function createWorkerUploadForm(worker: CfWorkerInit): FormData {
 		...(usage_model && { usage_model }),
 		...(migrations && { migrations }),
 		capnp_schema: bindings.logfwdr?.schema,
-		keep_bindings: ["plain_text", "json"],
+		...(keepVars && { keep_bindings: ["plain_text", "json"] }),
```

Comment:
I assume that not sending `keep_bindings` at all is valid? Have you tested this?
Is it safer to default to an empty array?

Reply:
Not sending `keep_bindings` is the old behaviour